CODEOWNERS

```
@@ -5,3 +5,4 @@
 # @pokealarm/dev-team - The PokeAlarm Team
 
 *	@pokealarm/dev-team
```

My understanding is that all the CODEOWNERS need to be in a single line, or else only the last one will be tagged for review: https://help.github.com/articles/about-code-owners/
```suggestion
*	@pokealarm/dev-team @KartulUdus
```